lib/http.js

```diff
@@ -8,7 +8,7 @@
 var req = require('request');
 var util = require('util');
 var url = require('url');
-var debug = require('debug')('node-soap');
+var debug = require('debug')('loopback:connector:http');
```

Comment:
The debug string should be `loopback:connector:soap` or `loopback:connector:soap:http`.